src/main/java/codebase/iterators/Iterators.java

```diff
@@ -1,6 +1,9 @@
 package codebase.iterators;
```

Comment:
This is the utility class for iterators. It does not make sense to have both Iterators and IteratorUtils.